[2022-10-18 18:04:07,290] {taskinstance.py:1159} INFO - Dependencies all met for <TaskInstance: clinical_trials__lung_cancer__etl__20220109.load_data_to_mongodb_task scheduled__2022-10-18T16:00:00+00:00 map_index=32 [queued]>
[2022-10-18 18:04:07,301] {taskinstance.py:1159} INFO - Dependencies all met for <TaskInstance: clinical_trials__lung_cancer__etl__20220109.load_data_to_mongodb_task scheduled__2022-10-18T16:00:00+00:00 map_index=32 [queued]>
[2022-10-18 18:04:07,302] {taskinstance.py:1356} INFO - 
--------------------------------------------------------------------------------
[2022-10-18 18:04:07,303] {taskinstance.py:1357} INFO - Starting attempt 1 of 1
[2022-10-18 18:04:07,304] {taskinstance.py:1358} INFO - 
--------------------------------------------------------------------------------
[2022-10-18 18:04:07,310] {taskinstance.py:1377} INFO - Executing <Mapped(_PythonDecoratedOperator): load_data_to_mongodb_task> on 2022-10-18 16:00:00+00:00
[2022-10-18 18:04:07,315] {standard_task_runner.py:52} INFO - Started process 2830 to run task
[2022-10-18 18:04:07,317] {standard_task_runner.py:79} INFO - Running: ['airflow', 'tasks', 'run', 'clinical_trials__lung_cancer__etl__20220109', 'load_data_to_mongodb_task', 'scheduled__2022-10-18T16:00:00+00:00', '--job-id', '388', '--raw', '--subdir', 'DAGS_FOLDER/process_news_dags.py', '--cfg-path', '/tmp/tmpwh_85f37', '--map-index', '32', '--error-file', '/tmp/tmpqyxdmnhp']
[2022-10-18 18:04:07,319] {standard_task_runner.py:80} INFO - Job 388: Subtask load_data_to_mongodb_task
[2022-10-18 18:04:07,330] {logging_mixin.py:115} WARNING - /root/.cache/pypoetry/virtualenvs/***-tXitOD5F-py3.8/lib/python3.8/site-packages/airflow/configuration.py:525 DeprecationWarning: The sql_alchemy_conn option in [core] has been moved to the sql_alchemy_conn option in [database] - the old setting has been used, but please update your config.
[2022-10-18 18:04:07,358] {task_command.py:370} INFO - Running <TaskInstance: clinical_trials__lung_cancer__etl__20220109.load_data_to_mongodb_task scheduled__2022-10-18T16:00:00+00:00 map_index=32 [running]> on host 6cdf6d517dc4
[2022-10-18 18:04:07,397] {taskinstance.py:1569} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_EMAIL=wais@datakimia.com,david@datakimia.com,juanm@datakimia.com,tomas@datakimia.com
AIRFLOW_CTX_DAG_OWNER=atacana
AIRFLOW_CTX_DAG_ID=clinical_trials__lung_cancer__etl__20220109
AIRFLOW_CTX_TASK_ID=load_data_to_mongodb_task
AIRFLOW_CTX_EXECUTION_DATE=2022-10-18T16:00:00+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2022-10-18T16:00:00+00:00
[2022-10-18 18:04:07,399] {process_news_dag_factory.py:128} INFO - Going to load data from clinical_trials about topic MK-7339
[2022-10-18 18:04:07,420] {db_helpers.py:35} INFO - Connected successfully!!!
[2022-10-18 18:04:07,424] {_universal.py:476} INFO - Request URL: 'https://atacananews.blob.core.windows.net/atacana-blob-container?restype=REDACTED&comp=REDACTED&prefix=REDACTED&delimiter=REDACTED'
Request method: 'GET'
Request headers:
    'x-ms-version': 'REDACTED'
    'Accept': 'application/xml'
    'User-Agent': 'azsdk-python-storage-blob/12.13.1 Python/3.8.15 (Linux-5.10.124-linuxkit-aarch64-with-glibc2.17)'
    'x-ms-date': 'REDACTED'
    'x-ms-client-request-id': '4253c1f6-4f0f-11ed-b07d-0242bebc0005'
    'Authorization': 'REDACTED'
No body was attached to the request
[2022-10-18 18:04:08,233] {_universal.py:504} INFO - Response status: 200
Response headers:
    'Transfer-Encoding': 'chunked'
    'Content-Type': 'application/xml'
    'Server': 'Windows-Azure-Blob/1.0 Microsoft-HTTPAPI/2.0'
    'x-ms-request-id': '7d3bc50b-a01e-000a-1c1c-e31bb0000000'
    'x-ms-client-request-id': '4253c1f6-4f0f-11ed-b07d-0242bebc0005'
    'x-ms-version': 'REDACTED'
    'Date': 'Tue, 18 Oct 2022 18:04:07 GMT'
[2022-10-18 18:04:08,236] {process_news_dag_factory.py:145} INFO - No blob found in path: lung-cancer/clinical-trials/20221018/mk-7339/
[2022-10-18 18:04:08,245] {taskinstance.py:1395} INFO - Marking task as SKIPPED. dag_id=clinical_trials__lung_cancer__etl__20220109, task_id=load_data_to_mongodb_task, map_index=32, execution_date=20221018T160000, start_date=20221018T180407, end_date=20221018T180408
[2022-10-18 18:04:08,280] {local_task_job.py:156} INFO - Task exited with return code 0
[2022-10-18 18:04:08,324] {local_task_job.py:273} INFO - 0 downstream tasks scheduled from follow-on schedule check
